### ISSUE TITLE
[BUG] Issue with furniture storage mechanic

### ISSUE THREAD

**Issue by BiallyDEV** [oraxen/oraxen#576]
### I have checked...

- [X] I am using the latest version of Oraxen
- [X] I am using the latest version of any dependencies
- [X] I have checked if any similar bug reports exist

### Description

https://paste.shockbyte.com/kejihupabodimemuzoja
After adding to furniture custom storage it breaks furniture and didin't destroy furniture and barriers.

### Steps to reproduce

Just add even example of storage mechanic to furniture.

### Plugin Version

1.148.0

### ProtocolLib Version

ProtocolLib version 5.0.0-SNAPSHOT-b602

### Server Version

Paper version git-Paper-285 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: b8919a7)

### How breaking is the bug?

Minor Bug - Plugin completely functional, but features have non-working aspects

**Comment by Boy0000** [comment]
Did you add it to an already configured furniture? @BiallyDEV 
Like:

1. make a furniture
2. place it in your server
3. add storage mechanic to the config
4. try and break it ingame

**Comment by BiallyDEV** [comment]
@Boy0000

1. Create a furniture without storage with multiple barriers

**Comment by BiallyDEV** [comment]
TV

**Comment by BiallyDEV** [comment]
@Boy0000 

1. I created a furniture with multiple barriers without storage mechanic, test it if everything work, it's work, it's break after destroying any of barrier, every barrier got removed.

2. Then i remove this furniture from the world and add a storage mechanic, restart server, after restart take this furniture from /o inv and place it. 

3. Storage mechanic work beside open sound and after destroying any of barrier the furniture still be here and rest of barriers as well.
